React/Base/RCTBridge.m

```diff
@@ -27,7 +27,8 @@
 NSString *const RCTJavaScriptDidLoadNotification = @"RCTJavaScriptDidLoadNotification";
 NSString *const RCTJavaScriptDidFailToLoadNotification = @"RCTJavaScriptDidFailToLoadNotification";
 NSString *const RCTDidInitializeModuleNotification = @"RCTDidInitializeModuleNotification";
-NSString *const RCTDidSetupModuleNotification = @"RCTDidSetupModuleNotification";
+NSString *const RCTDidSetupNativeModuleNotification = @"RCTDidSetupNativeModuleNotification";
+NSString *const RCTDidSetupTurboModuleNotification = @"RCTDidSetupTurboModuleNotification";
```

Comment:
A module is either a NativeModule or a TurboModule (and you know which is which). So I'm not sure why it's necessary to to break this notification down into two. Eventually, we plan to remove the Bridge and its accompanying NativeModule system, so we'll have to undo this change anyway. Thoughts?

Reply:
Good shout. It really didn't make sense to do so when I looked at this for the second time.